packages/pipeline-editor/src/PipelineEditor/index.tsx

```diff
@@ -572,7 +572,14 @@ const PipelineEditor = forwardRef(
       if (isNodeTipEvent(tipType, e) && e.node.type === "execution_node") {
         const error = e.node.app_data.invalidNodeError;
         const properties = controller.current.properties(e.node.id);
-        return <NodeTooltip error={error} properties={properties} />;
+        const node: any = controller.current.getPaletteNode(e.node.op);
```

Comment:
does this need to be `any`?

Reply:
It was typed as `unknown` so I couldn't access `node.label` - I just pushed a change that uses the typed `getAllPaletteNodes` function instead